pandas/tests/arithmetic/test_period.py

```diff
@@ -572,8 +572,8 @@ def test_ops_frame_period(self):
                 "B": [Period("2015-05", freq="M"), Period("2015-06", freq="M")],
             }
         )
-        assert df2["A"].dtype == "Period[M]"
-        assert df2["B"].dtype == "Period[M]"
+        assert df2["A"].dtype == "period[M]"
+        assert df2["B"].dtype == "period[M]"
```

Comment:
can these stay as they were? let's try to reduce the diff for future reviewers

Reply:
Thanks, I missed that. I rolled my changes here back.